docs/stencil-docs/customizing-checkout/optimized-one-page-checkout.md

```diff
@@ -44,11 +44,11 @@ Within the <span class="fn">optimized-checkout.scss</span> file, to set up Optim
     }
 ```
 
-## Classes Available for Customization
+## Classes available for customization
 
 Below are the classes provided in Cornerstone's <span class="fn">optimized-checkout.scss</span>, with the corresponding page elements that they style. _These class names and mappings are subject to change, so please check the inline comments in the <span class="fn">optimized-checkout.scss</span> file that you downloaded with your current Cornerstone release._
 
-| **Class** | **Styles This Optimized One-Page Checkout Element:** |
+| **Class** | **Styles this Optimized One-Page Checkout element:** |

```

Comment:
Should Optimized Checkout be all lowercase? Or should it be capitalized?

Reply:
In this case, I thought it should be capitalized because it is a name. The other occurrences were descriptions or adjectives.